src/supporting-data/keywords/components/results/KeywordsCard.tsx

```diff
@@ -33,7 +33,9 @@ const KeywordsCard = ({ data }: { data: KeywordsAPIModel }) => {
       header={
         <>
           <CardCheckboxCell id={id} />
-          <h2 className="tiny">{data.keyword.name}</h2>
+          <h2 className="tiny">
+            {data.keyword.name} ({data.keyword.id})
+          </h2>
```

Comment:
Is this needed here though?
I understand that on the entry page it's to match the current website, but here it's not displayed in the result page on the current website.
Also, I'm not sure how useful that is for a user (but I don't know how this page is used), so I wouldn't go further than just matching the current website does.

Reply:
I had added for consistency as it was requested in the table. But I guess it doesn't really matter, so removed it.